akka-http-core/src/main/scala/akka/http/impl/util/ByteStringParserInput.scala

```diff
@@ -23,7 +23,7 @@ import akka.util.ByteString
 final class ByteStringParserInput(bytes: ByteString) extends DefaultParserInput {
   override def charAt(ix: Int): Char = (bytes(ix) & 0xFF).toChar
   override def length: Int = bytes.size
-  override def sliceString(start: Int, end: Int): String = bytes.slice(start, end - start).decodeString(StandardCharsets.ISO_8859_1)
+  override def sliceString(start: Int, end: Int): String = bytes.slice(start, end).decodeString(StandardCharsets.ISO_8859_1)
```

Comment:
If a bug comes in it's all our team's fault, slipped through review – don't let it get you down :-)
And totally agreed, bugfix is always good – look forward, keep hakking 😉